examples/tests/valid/api.w

```diff
@@ -0,0 +1,9 @@
+bring cloud;
+
+let api = new cloud.Api();
+
+let handler = inflight (message: str): str => {
+  return "Hello World";
+};
+// failed when using '/' at the beginning of the path
+api.get("hello", handler);
```

Comment:
I think the public API should expect a slash at the beginning. Even though it doesn't add any extra information per se, it improve readability and helps with maintaining code (imagine someone grepping for "/hello" in their codebase). It's also pretty common in other REST API libraries I've tried (expressjs, flask).

If needed, the individual backends (sim.Api, tfaws.Api) can strip it out so the correct terraform is generated.

```suggestion
api.get("/hello", handler);
```

Reply:
This is now solved.